### ISSUE TITLE
Exporting and importing SymmetricDiffeomorphicRegistration outputs

### ISSUE THREAD

**Issue by KevinAquino** [dipy/dipy#3361]
Hi,

I would like to export the output fields from the outputs from SymmetricDiffeomorphicRegistration and then re-import them so I don't have to keep doing this, is there a way this works?

For example I perform this action:

synDR = SymmetricDiffeomorphicRegistration(metric, SyNLevels)

and would like 

export_synDR(synDR,outputfile)

then 
synDR = load_synDR_output(outputfle)

Does this exist? if not how can I design this myself?

**Comment by KevinAquino** [comment]
I didn't look hard enough - I found it after searching the issues a little more!

For those that are interested:

```python
from dipy.align import write_mapping,read_mapping
write_mapping(synMapping,'outputField.nii.gz')

synMapping = load_synDR_output('outputField.nii.gz')
```

**Comment by KevinAquino** [comment]
I'm leaving this open - should this be added to https://workshop.dipy.org/documentation/1.7.0/examples_built/20_registration/affine_registration_3d/#sphx-glr-examples-built-20-registration-affine-registration-3d-py ? 

just a small little section at the end - this is an extremely useful feature!

**Comment by skoudoro** [comment]
Yes, we should add it and also add it in the recipes page.  Thank you for your feedback. I am keeping this issue open

**Comment by arokem** [comment]
@KevinAquino : if you have bandwidth to make a PR with the addition of this documentation, we can get it in pretty quickly!

**Comment by KevinAquino** [comment]
Hi @arokem @skoudoro ,

can do! can you tell me where the webpages live? I can create a PR :)

**Comment by skoudoro** [comment]
here: https://github.com/dipy/dipy/blob/master/doc/examples/affine_registration_3d.py

and it would be great to add a small note here also: https://github.com/dipy/dipy/blob/master/doc/recipes.rst

Thank you @KevinAquino !
